.github/ISSUE_TEMPLATE/use-case.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Use Case
+about: Use this template to propose an LWS use case.
+title: "[UC] <<brief description of use case>>"
+labels: triage, usecase
+assignees: ''
+
+---
+
+**As a [primary actor],**
+**I want [an action or feature],**
+**So that [a reason or benefit].**
+
+## Preconditions:
+
+*What preconditions do we expect there to be fulfilled prior to this use case being triggered?*
+
+## Trigger:
+
+*What triggers this use case?*
+
+## Actors:
+
+*Describe the primary actor, and any other relevant actors involved in this use case*
+
+## Distinction:
+
+*What is the challenge in realizing this use case? What distinguishes it from other, similar use cases?*
```

Comment:
My reading of "challenge in realizing" that it is either unclear here or works with some knowledge of potential solutions or short-comings of solutions. Better to connect it with distinguishing factors

```suggestion
*What unique challenges or distinguishing factors, like technical issues, user experience needs, or workflow integration, are associated with this use case?*
```

Alternatively drop the ", like..integration, " above.

Reply:
```suggestion
*What unique challenges or distinguishing factors (like technical issues, user experience needs, workflow integration, etc.) are associated with this use case?*
```